docs/api.rst

```diff
@@ -46,6 +46,7 @@ Making Models from PDFs
    ~workspace.Workspace
    ~patchset.PatchSet
    ~patchset.Patch
+   simplemodels.hepdata_like
```

Comment:
The built docs incorrectly classify `hepdata_like` as a class: https://pyhf.readthedocs.io/en/docs-add-simplemodel-docs/_generated/pyhf.simplemodels.hepdata_like.html @kratsg do you know how to fix this? Or should this get moved to the top level API docs?

Reply:
this is because of the template used.

Reply:
@kratsg Okay. Are you okay with the docs being slightly wrong here? Or do you want me to move it someplace else where the `modifierclass.rst` template isn't used?

Reply:
we need to start probably removing the template in some cases.

Reply:
Opened #943 for this.